Library/Homebrew/rubocops/urls.rb

```diff
@@ -217,6 +217,13 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
             problem "Use /archive/ URLs for GitHub tarballs (url is #{url})."
           end
 
+          archive_ref_tags_gh_pattern = %r{https://.*github.*/archive/(?![a-fA-F0-9]{40})(?!refs/tags/).*\.tar\.gz$}
+          audit_urls(urls, archive_ref_tags_gh_pattern) do |_, url|
+            next if url.end_with?(".git")
+
+            problem "Use /archive/refs/tags URLs for GitHub tarballs (url is #{url})."
```

Comment:
Would be nice to write an autocorrect for this if possible?

Reply:
any example of doing autocorrect?

Reply:
https://github.com/Homebrew/brew/blob/121fc592cf4452040ffd877299ecfbb2b4fe86aa/Library/Homebrew/rubocops/homepage.rb#L46

Can wait till a follow-up PR but, if it's not too hard: autocorrects are always nice to have.

Reply:
👍